### PR TITLE
readme

### PR DESCRIPTION
**Pull request by boaglio**
As you requested Readme with picture, installation guide and controls.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
-# snakeGame
-/* 🐍 🐍 🐍 🐍 🐍 */
 
-cool snake game with moveable food
+Welcome to snakeGame!
+===================
+A cool snake game with running food
+
+![alt text](https://github.com/srbcheema1/snakeGame/raw/master/src/images/picture1.png)
```
Comment (srbcheema1) [inline README.md]: image is not shown

File: README.md [hunk 2]
```diff
@@ -1,4 +1,28 @@
-# snakeGame
-/* 🐍 🐍 🐍 🐍 🐍 */
```
Comment (srbcheema1) [inline README.md]: try not to remove this . 
this looks cute :)

File: README.md [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
+Welcome to snakeGame!
+===================
+A cool snake game with running food
+
+![alt text](https://github.com/srbcheema1/snakeGame/raw/master/src/images/picture1.png)
+
+----------
+
+
+## Running
```
Comment (srbcheema1) [inline README.md]: remove this section running

File: README.md [hunk 4]
```diff
@@ -1,4 +1,28 @@
...
+
+>  git clone https://github.com/srbcheema1/snakeGame.git    
+>  cd snakeGame  
+>  ant run  
+
+## Controls
```
Comment (srbcheema1) [inline README.md]: this is a nice addition :+1:

**Review by srbcheema1 — COMMENTED** [review]
plz reactify it and send the PR again i am eager to merge it

**Comment by srbcheema1** [comment]
@boaglio 
also dont make so much commits , they cause problems in reading the history

File: README.md (hunk 1 above)
Comment (boaglio) [reply README.md]: It will be after merge. Take a look: https://github.com/boaglio/snakeGame/blob/master/src/images/picture1.png

**Comment by boaglio** [comment]
Ok, fixed as you requested.

**Comment by srbcheema1** [comment]
still there are lot of commits in PR .
you can do one thing . 
just copy the readme file and pic to another folder .
delete the branch you have made .
create a new branch .
add changes .
commit changes in one commit .
then create a PR 
these are good practices of making a PR :)

**Pull request closed without merging**